CMakeLists.txt

```diff
@@ -63,6 +63,20 @@ target_link_libraries(relay
   relay_node
 )
 
+ament_auto_add_library(throttle_node SHARED
+  src/throttle_node.cpp
+)
+
+rclcpp_components_register_nodes(throttle_node "topic_tools::ThrottleNode")
```

Comment:
I wonder if it would be possible to build another rclcpp_components utility for "main with logic" to encapsulate this pattern that we're repeating, like the `rclcpp_components_register_node(target PLUGIN "plugin" EXECUTABLE "executable_name")`

Non-blocking wondering though :)

Reply:
I think it is better to use cmake macro to create executable, but I don't have any good idea to process different arguments with `configure_file`.